controllers/release/adapter.go

```diff
@@ -729,7 +890,7 @@ func (a *adapter) validateProcessingResources() *controller.ValidationResult {
 		return &controller.ValidationResult{Err: err}
 	}
 
-	if releasePlan.Spec.Pipeline == nil {
+	if releasePlan.Spec.TenantPipeline == nil && releasePlan.Spec.FinalPipeline == nil {
```

Comment:
I don't think this line should have the && releasePlan.Spec.FinalPipeline == nil

This check is basically saying if the RP does not define a tenant pipeline, it must define a managed one, so make sure everything we need to run a managed pipelineRun exists. The final pipeline doesn't really factor into this logic if we are going by "you have to run a tenant *or* managed pipeline per release, just a final is not good enough"